docs/scenarios/solution-authoring/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,130 @@
+
+Prerequisites
+-------------
+
+- [Visual Studio 2015 Update 3 RC](https://www.visualstudio.com/downloads/visual-studio-prerelease-downloads#sec1) or higher
+- [.NET Core tools for Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- NuGet Visual Studio extension [v3.5.0-beta](https://dist.nuget.org/visualstudio-2015-vsix/v3.5.0-beta/NuGet.Tools.vsix) or later
+
+Creating a library targeting .NET Core
+--------------------------------------
+
+1. In the Visual Studio menu bar, choose **File** | **New** | **Project** and select **Class Library (Portable)**
+
+  ![New Project](../../../images/new-project-dialog-class-library-portable.png)
+
+2. Choose a name and location for your project and click **OK**
+
+3. The "Add Portable Class Library" dialog will appear.  Select **.NET Framework 4.6** and **ASP.NET Core 1.0** as targets and click **OK**
+
+  ![Portable targets dialog](../../../images/pcl-targets-dialog-net46-aspnetcore10.png)
+
+4. In Solution Explorer, right click on your project and choose **Properties**
+5. In the **Library** tab of the project properties, click on the **Target .NET Platform Standard** link, and click **Yes** in the dialog that is shown
+6. Open the `project.json` file in your project, and make the following changes:
+    - Change the version number of the `NETStandard.Library` package to `1.5.0-rc2-24027` (this is the .NET Core RC2 version of the package)
+    - Add the below `imports` definition inside the `netstandard1.5` framework definition.  This will allow your project to reference .NET Core compatible 
+NuGet packages that haven't been updated to target .NET Standard
+
+        ```json
+        "netstandard1.5": {
+            "imports": [ "dnxcore50", "portable-net452" ]
+        }
+        ```
+
+Creating a .NET Core console application
+----------------------------------------
+Building a console application for .NET Core requires some customization of the MSBuild build process.  You can find a sample project for a .NET Core console
+application called [CoreApp](https://github.com/dotnet/corefxlab/tree/master/samples/NetCoreSample/CoreApp) in the
+[corefxlab](https://github.com/dotnet/corefxlab) repo.  Another good option is to start with [coretemplate](https://github.com/mellinoe/coretemplate),
+which uses separate MSBuild targets files to target .NET Core instead of putting the changes directly in the project file.  
+
+It is also possible to start by creating a project in Visual Studio and modify it to target .NET Core.  The instructions below show the minimal steps
+to get this working.  In contrast to [CoreApp](https://github.com/dotnet/corefxlab/tree/master/samples/NetCoreSample/CoreApp) or
+[coretemplate](https://github.com/mellinoe/coretemplate), a project created this way won't include configurations for targeting Linux and OS X.
+
+Creating a .NET Core console application from Visual Studio
```

Comment:
> is this a standalone section or a subsection of Creating a .NET Core console application?

It would be a subsection, but looking at it now I don't think the heading is necessary, so I've removed it.